classes/games.rb

```diff
@@ -1 +1,12 @@
-# games
+require 'item'
+require 'date'
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at
+
+  def initialize(publish_date, multiplayer, last_played_at)
+    super(publish_date)
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+  end
+end
```

Comment:
I think you forgot the `can_be_archived` method to implement it.
![Screenshot 2023-07-05 130728](https://github.com/ShiroYaksha90/catalog-of-my-things/assets/122348996/deb9433c-16c2-4ed9-9b97-832868341f28)

Reply:
Hello @mahabubx7,

Thank you for your review and feedback. I appreciate your input. I wanted to let you know that I have taken note of your suggestions, and I will be implementing the methods as discussed in the next pull request, which will be labelled, "Implement Methods, #20."

Reply:
oh... my bad!